app/e2e/auth.setup.ts

```diff
@@ -11,8 +11,8 @@ const authMethodIsBasic = async (apiURL: string) => {
 };
 
 const getBasicCredentials = () => {
-  const user = process.env.PACKIT_E2E_BASIC_USER;
-  const password = process.env.PACKIT_E2E_BASIC_PASSWORD;
+  const user = "resideUser@resideAdmin.ic.ac.uk";
+  const password = "password";
```

Comment:
Why change these? These are using environment variables so that you can potentially run e2e test against non-local packit instances that potentially don't have these packit users. Currently our instances use github auth, but it would be good to be able to support non-local basic auth e2e tests.

Reply:
OMG i wasn't supposed to commit this!!!!! good catch thanks I've reerted